bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -352,4 +353,10 @@ static long getLongVariableOrDefault(ServerConfiguration conf, String keyName, l
             return conf.getLong(keyName);
         }
     }
+
+    @Override
+    public OfLong getListOfEntriesOfLedger(long ledgerId) throws IOException {
```

Comment:
If you don't implement this, please create a github issue and comment it here.

Reply:
created https://github.com/apache/bookkeeper/issues/2078 and adding comment in the code